airflow/www/views.py

```diff
@@ -2896,6 +2897,49 @@ def action_muldelete(self, items):
         self.update_redirect()
         return redirect(self.get_redirect())
 
+    @action('mulcopy', 'Copy', 'Are you sure you want to Duplicate selected connections?', single=False)
+    @provide_session
+    @auth.has_access(
+        [
+            (permissions.ACTION_CAN_CREATE, permissions.RESOURCE_CONNECTION),
+        ]
+    )
+    def action_mulcopy(self, connections, session=None):
+        """Copy multiple connections"""
+        for selected_con in connections:
```

Comment:
It might make sense to change `selected_con` to `selected_conn` since 'connection' is abbreviated as 'conn' in the rest of your code.